documentation/business-analyst-activities.md

```diff
@@ -1 +1,17 @@
 # Business Analyst Activities
+
+**How much detail should we go into with regards to setting up GITHUB account?
+
+
+## Updating Polygon Mappings in the mapped_locations.yml file
+This section details the process for updating the search mappings so that a polygon search is used in place of a point search.
+
+1. Navigate to the "Code" tab within github and press "go to file"
+2. Search for "mapped_locations.yml" and click on the file.
+  * This file contains a list of all of the mappings that have been applied, in alphabetical order.
+3. Navigate to the desired row, and input your new mapping in the following format:
+  * -['location that is searched by user', 'name of polygon we want to return']
+  * The automated tests are useful for confirming that all rows have been entered correctly. If one of these tests fails check that all "'" and "," have been entered correctly.
```

Comment:
Where are these? I didn't realise this could be done, so potentially someone else might not either.

Reply:
On your PR, in the merging section at the bottom there is a bit dedicated to the automated checks. This would show any checks that may have failed as part of any changes.

<img width="779" alt="Screenshot 2021-01-07 at 17 59 09" src="https://user-images.githubusercontent.com/72141/103927287-0d570500-5112-11eb-902d-5d36a1c7e10a.png">

Reply:
Have added image to the file, and a line about where this can be found.